libs/commons/Int64_ops.mli

```diff
@@ -0,0 +1,23 @@
+(* This module exists to be opened locally, so that normal arithmetic
```

Comment:
I can't believe Int64 does not define those operators ...

Reply:
Maybe we could rename to Int64_.ml like we have Set_.ml, 
and include Int64 from it + our extended operators